npm/src/upload/upload-form.ts

```diff
@@ -0,0 +1,47 @@
+import { TdrFile } from "@nationalarchives/file-information"
+import { stringType } from "aws-sdk/clients/iam"
```

Comment:
Not sure the `stringType` import is used?

Reply:
Correct, I'll remove it.